src/schema/artist/index.js

```diff
@@ -544,7 +672,12 @@ export const ArtistType = new GraphQLObjectType({
           },
           sort: SaleSorts,
         },
-        resolve: ({ id }, options, _request, { rootValue: { relatedSalesLoader } }) =>
+        resolve: (
+          { id },
+          options,
+          _request,
+          { rootValue: { relatedSalesLoader } }
```

Comment:
I agree this kind of formatting is annoying but soon we should sweep over and refactor everything so that the loader is pulled from the first slot rather than the last, allowing us to eliminate all of the additional arguments.